### ISSUE TITLE
Several elements under Simulator are not rendered as Focused state under High Contrast Off

### ISSUE THREAD

**Issue by Helen09** [microsoft/pxt-ev3#535]
**Repro steps:**
• Open https://makecode.mindstorms.com/beta#editor
• Keep High Contrast off
• Navigate by Tab key, try to reach the five buttons under Simulator

**Expected result:**
The five buttons under Simulator should be rendered as Focused once selecting it. 
![compare2](https://user-images.githubusercontent.com/23466737/39163264-459a5756-4769-11e8-836d-162c1ebb5e50.jpg)

**Actual result:(Edge, Firefox&Chrome)**
The five buttons are not rendered as Focused state. But it works when High Contrast On. 
![issue1](https://user-images.githubusercontent.com/23466737/39163270-4b7a6922-4769-11e8-97aa-ae8301fc5f70.jpg)

**Additional info**:
 LEGO® MINDSTORMS® Education EV3 version: 0.1.27
 Microsoft MakeCode version: 3.10.10

**Browser info:**
 Edge: 42.17134.1.0
 Firefox: 59.0.2
 Chrome: 66.0.3359.117
